compiler/luci/pass/src/FuseBCQPass.cpp

```diff
@@ -435,6 +468,8 @@ bool FuseBCQPass::run(loco::Graph *g)
   const int32_t start_magicnum = -2e9 + 27;
   const int32_t end_magicnum = 2e9 - 27;
 
+  loco::Graph *main_graph = g;
```

Comment:
Q) why is `main_graph` introduced? `g` is also used below.

Reply:
Another PRs will be created to substitute `run(loco::Graph *g)` to `run(luci::Module *m)`.
In those situation, multiple subgraphs will be appeared and `main_graph` is the main graph which includes BCQ information :)